helpdesk/templates/helpdesk/ticket.html

```diff
@@ -185,7 +192,7 @@ <h3>{% trans "Respond to this ticket" %}</h3>
         </div></dd>
         {% endifequal %}
 
-        {% if helpdesk_settings.HELPDESK_UPDATE_PUBLIC_DEFAULT %}
+        {% if helpdesk_settings.HELPDESK_UPDATE_PUBLIC_DEFAULT or not is_staff %}
```

Comment:
Feel like this should be an AND.  Whatever the setting is, but then still don't show it to non staffers.

Reply:
I think you're right, (that this should be "AND is staff" rather than "OR NOT is_staff" but I'm a little confused because I thought I saw this radio button present before the branch and then not present after testing with the branch. (But I may also not be thinking quite straight as I'm on little sleep at this point...)

So, tell me what you see in testing.